librz/core/cconfig.c

```diff
@@ -3300,7 +3300,7 @@ RZ_API int rz_core_config_init(RzCore *core) {
 	p = rz_sys_getenv("RZ_CFG_OLDSHELL");
 	SETCB("cfg.oldshell", p ? "true" : "false", &cb_oldshell, "Use old radare2 parser");
```

Comment:
Should be this removed too then? We removed most of the oldshell code anyway.

Reply:
true, good point.